### PR TITLE
check function: check that all the parameters are right before sendin…

### PR DESCRIPTION
**Pull request by CarlosHL**
…g the survey to the back end. Other minor changes to Brady's code.

### CODE REVIEW

File: static/javascripts/components/survey_card.jsx [hunk 1]
```diff
@@ -67,9 +93,8 @@ var SurveysPage = React.createClass({
             item_title: this.state.item_title,
```
Comment (CarlosHL) [inline static/javascripts/components/survey_card.jsx:73]: @sungbae this field

Comment (sungbae) [inline static/javascripts/components/survey_card.jsx:73]: @CarlosHL Ah I see, but I'm a little confused. In the getinitialstate() function, we initialize a field called "item_name". Isn't "item_name" the actual title of the survey? Then you have some function called updateTitle(), and this updates a new field called "item_title". Shouldn't that be "item_name" instead?

**Comment by Brady76** [comment]
@CarlosHL you have some merge conflicts, but your submission checking is really great.

File: static/stylesheets/styles.css [hunk 2]
```diff
@@ -1,479 +0,0 @@
-/**
- * Copyright 2015 Google Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (Brady76) [inline static/stylesheets/styles.css:4]: Why'd you delete this? It's causing issues in your PR

File: static/javascripts/components/survey_card.jsx [hunk 3]
```diff
@@ -288,6 +316,18 @@ var CheckboxQuestion = React.createClass({
     addOption: function(){
         var options = this.state.options;
         var numOptions = this.state.numOptions;
+
+        if (numOptions>0){
+            for(var i=0; i<options.length;i++){
+              if (options[i].title==""){
+              alert("Complete all the current options before adding one");
```
Comment (Brady76) [inline static/javascripts/components/survey_card.jsx]: I don't know if I think this is necessary. When I make surveys I like to click the button till I have enough options and then fill it out. It just seems inconvenient without great reason.

File: static/javascripts/components/survey_card.jsx [hunk 4]
```diff
@@ -73,9 +101,8 @@ var SurveysPage = React.createClass({
             item_title: this.state.item_title,
             questions : questions
         };
-        console.log(surveyObj);
-
 
+        var test = this.checkSurvey(surveyObj);
```
Comment (Brady76) [inline static/javascripts/components/survey_card.jsx]: Does this even stop the ajax from posting? I might not know how alerts work but wouldn't we want to say if(test==true){ajax post}?

File: static/stylesheets/styles.css (hunk 2 above)
Comment (CarlosHL) [inline static/stylesheets/styles.css:4]: that's weird. I checked that it was the wrong one... any way we should have only one css.

File: static/javascripts/components/survey_card.jsx (hunk 3 above)
Comment (CarlosHL) [inline static/javascripts/components/survey_card.jsx]: I think is the easiest way to check the stuff so the user does not miss anything.

File: static/javascripts/components/survey_card.jsx (hunk 4 above)
Comment (CarlosHL) [inline static/javascripts/components/survey_card.jsx]: Good point, going to change this asap.

**Pull request closed without merging**